zk-token-sdk/src/encryption/elgamal.rs

```diff
@@ -76,6 +76,8 @@ pub enum ElGamalError {
     DerivationMethodNotSupported,
     #[error("seed length too short for derivation")]
     SeedLengthTooShort,
+    #[error("seed length too long for derivation")]
+    SeedLengthTooLarge,
```

Comment:
and same here, "large" -> "long"
```suggestion
    SeedLengthTooLong,
```